### PR TITLE
2021 feature dump

### PR DESCRIPTION
**Pull request by sampollard**
Here are a bunch of changes I made throughout. The main ones are
- Removing the requirement of APA citation style
- Clarifying documentation
- Renaming files to not imply APA
- Updating and simplifying the Makefile and build system
- Adding a feature for a "list of source code listings" in the same style as "list of figures"
- Updating the graduate school name and dean for 2021
- Removing unused packages

There is also
- Deleting trailing whitespace
- Fixing typos

### CODE REVIEW

File: latex_files/Makefile [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+	touch $(MAIN).tex
+	$(LATEX) $(LATEXFLAGS) $(MAIN).tex
+	$(LATEX) $(LATEXFLAGS) $(MAIN).tex
+	$(LATEX) $(LATEXFLAGS) $(MAIN).tex
 
 short: *.tex 
-	pdflatex main.tex
+	$(LATEX) $(LATEXFLAGS) $(MAIN).tex
+
+findref:
```
Comment (jglev) [inline latex_files/Makefile:18]:
```suggestion
findref: all
```

Would it be useful to call out the `all` target here, as in the above?

**Review by jglev — COMMENTED** [review]
@sampollard, this looks good to me overall! I have two comments, plus a question:

1. The [`LICENSE` file](https://github.com/sampollard/unofficial-uoregon-grad-school-dissertation-latex-markdown-apa-format/blob/2021-feature-dump/LICENSE#L2) can be updated [like the Readme](https://github.com/publicus/unofficial-uoregon-grad-school-dissertation-latex-markdown-apa-format/pull/9/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R120-R121), if you like! : )
2. I appreciate the overhaul you've made to the Makefile! As you documented in the Readme, running `make` from within `latex_files` builds successfully for me. I do note that `./Markdown_to_LaTeX_PDF_Build_Script.sh` is not building successfully. That's not a blocker for merging this in my mind, though, because I also note that it seems to be failing to build in the `master` branch, as well (albeit in a different place).

For now, if you're open to it, I'd suggest adding a brief note to the Readme that that script (and `testing_script/Run_Tests_to_Make_Sure_that_Everything_Builds.sh`, I think) are example implementations and aren't currently functional. Ideally, down the road, both could get incorporated into the Makefile (e.g., adding a Make target that would convert each Markdown file into a `.tex` file and update the existing `.tex` files to import them), perhaps alongside some more robust tests. But that needn't happen here, in this PR, of course!
3. Since you're closer to the Graduate School than I am at this point (I defended in late 2016, and now live on the other side of the country), are you aware of anyone at the Graduate School, or any faculty, who might be interested in taking over ownership of this repo.? I'm fine to maintain it in my GitHub namespace, but I like to ask anyone who uses the repo., since I don't have as many up-to-date contacts at the University, and it would be ideal for someone closer to UO to be maintaining it, in my mind.

Again, thank you!

File: latex_files/Makefile (hunk 1 above)
Comment (sampollard) [reply latex_files/Makefile:18]: No, since that is a circular dependency; I'm saying here that `findref` runs whenever you try to build `$(MAIN).pdf`.

**Comment by sampollard** [comment]
Glad it looks useful!

I didn't work on the markdown part because I think that is a "special" use case. That is, I think most people will _not_ use the markdown conversion (that is why I also renamed the files to not be `.md`. I've heard people have great success using e.g. RMarkdown or Sphinx to generate their entire Ph.D. dissertations but I think it's enough work making sure the plain LaTeX is compatible. Regardless, I'll add a note. Maybe leaving it in is helpful for people using RMarkdown or pandoc or whatever LaTeX generator.

Let me look into `testing_script/Run_Tests_to_Make_Sure_that_Everything_Builds.sh` - that is a good sort of unit test to have working.

Regarding ownership, I can't think of anyone. I'm happy to be the owner of it for a while, and maybe the next person who makes a MR can get the buck passed on to him/her :)

File: latex_files/Makefile (hunk 1 above)
Comment (jglev) [reply latex_files/Makefile:18]: Ah, I should have read the full file more closely; I see what you mean now. Thank you for explaining!

**Comment by jglev** [comment]
> I think most people will not use the markdown conversion

I agree; at least, that's my suspicion, too.

> Regardless, I'll add a note. Maybe leaving it in is helpful for people using RMarkdown or pandoc or whatever LaTeX generator.

Thank you! I wrote my own diss. using RMarkdown, so I'd at least like to keep some sort of reference implementation for future users in similar situations. Though perhaps even a description of an example workflow in the Readme ("Write a script that converts individual markdown files to TeX using Pandoc, etc.") may be enough to get future users going.

> I'm happy to be the owner of it for a while, and maybe the next person who makes a MR can get the buck passed on to him/her :)

That sounds great to me! And that was really the original idea, back when Tyler Matta shared the LaTeX work that he'd done for his own dissertation with me (after he'd expanded some other prior candidate's work, as I remember). I'm totally happy with that, and appreciate your willingness! I'll transfer ownership, and open my own MR updating the Readme : )

**Comment by jglev** [comment]
I've created a new GitHub organization to facilitate transferring ownership now and in the future, and moved this repo into its namespace. I've invited you as an admin, @sampollard; if/when you're comfortable accepting, you can feel free to update the org. email address and remove my admin access to the org.

**Comment by sampollard** [comment]
Hey, I updated the README to be updated with the filenames, also recovered the missing `5` file which would have bugged people (myself included).

I looked briefly at your Markdown scripts and I think fixing them would require a lot more work. I apologize for not writing my changes in a way which doesn't break the markdown, but maybe someone else will have the motivation to get this working.

**Comment by jglev** [comment]
> I looked briefly at your Markdown scripts and I think fixing them would require a lot more work. I apologize for not writing my changes in a way which doesn't break the markdown, but maybe someone else will have the motivation to get this working.

I agree that in a repo like this, accepting updates as whomever has the motivation to work on them is a fine way to proceed. So no worries!

**Pull request merged**